Assets/MixedRealityToolkit-Examples/Prototyping/Shaders/UnlitTransparentTriplanar.shader

```diff
@@ -1,8 +1,8 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 ///
-/// Simple vertex shader that blends static lighting and camera lighting.
```

Comment:
Still a lot of formatting changes in this file.

Could we revert these just to see the real changes please?

Reply:
These files are not related to this update - I have reverted changes on these.